scripts/cipublish

```diff
@@ -49,7 +49,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
             GIT_COMMIT="${GIT_COMMIT}" docker-compose \
                       -f "${DIR}/../docker-compose.yml" \
                       -f "${DIR}/../docker-compose.test.yml"\
-                      build nginx app-server tile-server
+                      build nginx api-server tile-server
 
             # Evaluate the return value of the get-login subcommand, which
```

Comment:
Oh, I think we may need to manually create the container repositories in the ECS dashboard to match up with the names